examples/game_of_life/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenCensus Game of Life Codelab
+
+This codelab demonstrates OpenCensus with a simple client/server Java example based on
+[Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)
+(abbreviated as Life for the remainder of this Codelab). You will learn how to define OpenCensus
+tags, measures and views, how to add tags to the Life application, how to record stats against
+current tag context, and how to view and understand the stats collected against those tags.
+
+The Life application used in this Codelab consists of a Life client and a Life server.
+
+The Life client consists of three two-dimensional matrices: 8x8, 16x16, and 32x32.
+Each of the cells in the matrices can be alive or dead depending on some rules.
+The client takes in the number of RPC calls from the user and sends that many requests to the server
+to calculate the alive/dead cell configuration for the matrices. The server performs the
+calculations and sends the response back to the client to display the
+results.
+
+## Prerequisites
+
+1. Install [Bazel](https://docs.bazel.build/versions/master/install.html).
```

Comment:
Suggestion: If you setup gradle for this project users don't need 1 and 2.

Reply:
Created https://github.com/songy23/game-of-life-demo/issues/1 to follow up. (Chose bazel first because it has built-in support for protoc.)